flixel/FlxG.hx

```diff
@@ -324,6 +324,7 @@ class FlxG
 	 * whatever floats your boat! For maximum performance try bundling a lot of objects
 	 * together using a FlxGroup (or even bundling groups together!).
 	 * NOTE: does NOT take objects' scrollfactor into account, all overlaps are checked in world space.
+	 * NOTE: for checking overlap with FlxTilemaps that contain empty tiles use FlxTilemap#overlaps() instead.
```

Comment:
Hm, still not a huge fan of this description.. We should explain the _why_, not just redirect to another part of the API without any explanation.